pontoon/base/static/js/fluent_interface.js

```diff
@@ -111,6 +111,23 @@ var Pontoon = (function (my) {
     return false;
   }
 
+  /*
+   * Is ast of a message without value and with 1 attribute,
+   * which only contains simple elements?
```

Comment:
Nit, I think those comments are better when they start with `Return`. In this case, something like `Return true when an AST has no value and a single attribute with only simple elements`, for example.